src/app/rosetta/test-agent/agent.ml

```diff
@@ -546,6 +582,15 @@ let check_new_account_user_commands ~logger ~rosetta_uri ~graphql_uri =
   [%log info] "Created payment and waited" ;
   (* Stop staking so we can rely on things being in the mempool again *)
   let%bind _res = Poke.Staking.disable ~graphql_uri in
+  (* TODO: Why does introducing this break everything?? *)
```

Comment:
Is this TODO done? If not, it would be good to add a little more info here.

Reply:
Yes, whoops